phpstan-baseline.neon

```diff
@@ -0,0 +1,27 @@
+parameters:
+	ignoreErrors:
+		-
+			message: "#^Parameter \\#1 \\$str of function sanitize_text_field expects string, array\\<string\\>\\|string given\\.$#"
```

Comment:
`sanitize_text_field` will return empty string `''` when provided an array.

We have validation after sanitising, so it's cleaner to ignore these errors for now.

Note: WP core does `sanitize_text_field( wp_unslash (` throughout the codebase